.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
+ignore =
+    # E126 - continuation line over-indented for hanging indent
+    E126,
+    # N813 - camelcase imported as lowercase
```

Comment:
I think there's already flake8 configuration in the `setup.cfg` file, could this config go in there as well?